t/chaos/utils.go

```diff
@@ -0,0 +1,192 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package chaos
+
+import (
+	"fmt"
+	"net/http"
+	"strconv"
+	"strings"
+	"time"
+
+	"github.com/gavv/httpexpect/v2"
+	. "github.com/onsi/gomega"
```

Comment:
Why use the period `.` here?

Reply:
It seems a convention to use `ginkgo`/`gomega` this way ([doc](https://onsi.github.io/ginkgo/#bootstrapping-a-suite)), and the doc also says [there are only rare cases to avoid this](https://onsi.github.io/ginkgo/#avoiding-dot-imports), so I guess that would be fine.